pkg/controller/job/job_controller_test.go

```diff
@@ -3694,7 +3708,9 @@ func TestWatchJobs(t *testing.T) {
 	stopCh := make(chan struct{})
 	defer close(stopCh)
 	sharedInformerFactory.Start(stopCh)
-	go manager.Run(context.TODO(), 1)
+	ctx, cancel := context.WithCancel(context.Background())
```

Comment:
Revert these changes, they are have nothing to do with the test. +1 for another PR for cleanup, but coupling unnecessary changes will make this important PR take more time.

Reply:
/Ack. Sorry, this was my bad. I had this change and got pushed into this un-intentionally. It was a stash from previous thing I was suspecting. Cleaning up